stable/grafana/templates/dashboards-import-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.dashboardImports.enabled -}}
+{{- if .Values.dashboardImports.files -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "grafana.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Values.server.name }}"
```

Comment:
Component should normally be hard-coded. The component and name suffix should match. In fact, components only make sense when there are multiple, which is not the case. I would remove the `component` labels and fix the `app` label to be `{{ template "grafana.name" . }}`. I know this is not an issue of your PR because it's already been like this. Maybe you'd like to create another PR to fix this?

Reply:
I'll create another PR for that.

Reply:
Thanks. I figured this is based on existing code.